utils/params.py

```diff
@@ -0,0 +1,2 @@
+def parse_boolean(parameter_value: str) -> bool:
+    return parameter_value.strip().lower() in {"1", "true", "on", "yes"}
```

Comment:
How about something like

```suggestion
from typing import Any

TRUE_VALUES = {"1", "true", "on", "yes", "y"}

def parse_boolean(parameter_value: Any) -> bool:
    if not parameter_value:
        return False
    return str(parameter_value).strip().lower() in TRUE_VALUES
```